### PR TITLE
Feature project activity log

### PR DESCRIPTION
**Pull request by AdityaKhatri**

### CODE REVIEW

File: src/views/Project/Details/General/Dashboard/index.js [hunk 1]
```diff
@@ -33,7 +33,12 @@ const propTypes = {
 
 const defaultProps = {
     className: '',
-    projectDashboard: {},
+    projectDashboard: {
```
Comment (tnagorra) [inline src/views/Project/Details/General/Dashboard/index.js]: leadActivity and entriesActivity should be checked for undefined values where it is used

File: src/utils/common.js [hunk 2]
```diff
@@ -20,3 +20,8 @@ export const pick = (obj, keys) => keys.reduce(
     (acc, key) => ({ ...acc, [key]: obj[key] }),
     {},
 );
+
+export const camelToNormalCase = (text) => {
```
Comment (tnagorra) [inline src/utils/common.js:26]: Fails for case like "IAmBADBoy"

Comment (tnagorra) [reply src/utils/common.js:26]: Also missing test for function in utils

File: src/components/ActivityLog/index.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    render() {
+        const {
+            log,
+            className,
+        } = this.props;
+
+        return (
+            <div className={`${styles.activityLog} ${className}`}>
+                <h3 className={styles.header}>
+                    Activity Log
```
Comment (tnagorra) [inline src/components/ActivityLog/index.js]: Every string should be translated

File: src/components/ActivityLog/index.js [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+export default class ActivityLog extends React.PureComponent {
+    static propTypes = propTypes;
+    static defaultProps = defaultProps;
+
+    static logItemKeySelector = l => l.key;
+
+    constructor(props) {
+        super(props);
+        this.state = {};
```
Comment (tnagorra) [inline src/components/ActivityLog/index.js]: Unused this.state

File: src/components/ActivityLog/LogItem/index.js [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+
+import AggregatedLogItem from '../AggregatedLogItem';
+import styles from './styles.scss';
+
+const LogItem = ({
+    user,
+    time,
+    fields,
+}) => {
+    if (Object.keys(fields).length > 1) {
```
Comment (tnagorra) [inline src/components/ActivityLog/LogItem/index.js:20]: A comment would help

File: src/utils/common.js (hunk 2 above)
Comment (AdityaKhatri) [reply src/utils/common.js:26]: no!

**Pull request merged**